Source/DataModels/Components/ComponentMeshRenderer.cpp

```diff
@@ -223,6 +226,13 @@ void ComponentMeshRenderer::DrawMaterial(Program* program) const
 
 		float3 viewPos = App->GetModule<ModuleCamera>()->GetCamera()->GetPosition();
 		program->BindUniformFloat3("viewPos", viewPos);
+		glActiveTexture(GL_TEXTURE8);
+		glBindTexture(GL_TEXTURE_CUBE_MAP, App->GetModule<ModuleScene>()->GetLoadedScene()->GetCubemap()->GetIrradiance());
```

Comment:
We can call `GetCubemap` only once and store the value.
```suggestion
		Cubemap* cubemap = App->GetModule<ModuleScene>()->GetLoadedScene()->GetCubemap();
		glBindTexture(GL_TEXTURE_CUBE_MAP, cubemap->GetIrradiance());
```